modules/multivcfanalyzer/main.nf

```diff
@@ -0,0 +1,72 @@
+process MULTIVCFANALYZER {
+    tag '$fasta'
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::multivcfanalyzer=0.85.2" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/multivcfanalyzer:0.85.2--hdfd78af_1':
+        'quay.io/biocontainers/multivcfanalyzer:0.85.2--hdfd78af_1' }"
+
+    input:
+    path vcfs
+    path fasta
+    path snpeff_results
+    path gff
```

Comment:
Don't you want a single tuple for the input, and separate path for referecnes?

Reply:
>Don't you want a single tuple for the input, and separate path for referecnes?

I don't follow, sorry. Could you rephrase? There is a separate path for fasta here?

Reply:
I'm assuming that vcfs and snpeff_results are your input, and gff and fasta your references.
so why not:
```
tuple path(vcfs), (snpeff_results)
path fasta
path gff
```

Reply:
ah I see. No - it's a single snpEff file as input.

Reply:
The real input is the vcfs though, and then other reference files (fasta, gff)...?

It is very confusing... main input are the VCFs of multiple files, but you can include the annotation files based on the reference you used...